app/android/app/src/dev/google-services.json

```diff
@@ -72,14 +124,14 @@
         "appinvite_service": {
           "other_platform_oauth_client": [
             {
-              "client_id": "366164701221-891alrejtv2s1eg0cbkov0ct3gga6hjh.apps.googleusercontent.com",
+              "client_id": "366164701221-2i3lvsf1b0b4ikju7vf598gnbdsj7n90.apps.googleusercontent.com",
               "client_type": 3
             },
             {
-              "client_id": "366164701221-fdv476f10fl969nd65dv97tfajqb9jr1.apps.googleusercontent.com",
+              "client_id": "366164701221-ad0lgt4d4gktqt0ejnasj3mpn36qrhs0.apps.googleusercontent.com",
               "client_type": 2,
               "ios_info": {
-                "bundle_id": "de.codingbrain.sharezone.app.dev"
+                "bundle_id": "de.codingbrain.sharezone.macos.dev"
```

Comment:
Where is this `de.codingbrain.sharezone.macos.dev` bundle id from? The macOS app has the bundle id `de.codingbrain.sharezone.app.dev`.

Reply:
Auto-generated from the `flutterfire configure` cli.  
Since the `google-services.json` is used in Android it shouldn't matter anyways.